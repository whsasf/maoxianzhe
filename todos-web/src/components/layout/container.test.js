import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Container from "./container";

it("should renders correctly", () => {
  const tree = renderer.create(<Container>somthing</Container>).toJSON();
  expect(tree).toMatchSnapshot();
});
