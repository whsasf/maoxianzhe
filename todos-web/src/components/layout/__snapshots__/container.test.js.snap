// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  background: #fff;
  padding: 24px;
  min-height: calc(100vh - 186px);
}

<div
  className="c0"
>
  somthing
</div>
`;
